backend/servicelists/TM-STREAM0075r1 DVB-DASH_Reference_Streams.xml

```diff
@@ -189,16 +189,16 @@
         <AccessibilityAttributes>
           <!-- WVTT -> contentType="text" codecs="wvtt" lang="en" -->
           <tva:SubtitleAttributes>
-            <tva:Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:99"/>  <!--other-->
+            <tva:Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:3"/>  <!--ISOBMFF-->
             <tva:Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:4"/>  <!--WebVTT-->
-            <tva:SubtitleLanguage>en</tva:SubtitleLanguage>
+            <tva:SubtitleLanguage>eng</tva:SubtitleLanguage>
             <tva:SuitableForTTS>false</tva:SuitableForTTS>
           </tva:SubtitleAttributes>
           <!-- TTML ->  contentType="text" lang="en" codecs="stpp.ttml.im1t"-->
           <tva:SubtitleAttributes>
             <tva:Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:2"/>  <!--DVB TTML-->
```

Comment:
I don't know if the values in this service lists are based on "real" streams. But it is unlikely that the TTML is carried in DVB-TTML as this is specified for TS transport. I think the value should be ISOBMFF. It would also be better to use the same coding format as in the samples (EBU-TT-D 1.0.1 and IMSC 1.0.1). This way the subtitles can also be displayed on HbbTV devices >= Version 2.0.4.

Reply:
The <Carriage> and <Coding> elements are based on the information in the DASH MPD.
If you look at the service `tag:dvb.org,2020:CMAF-3`  (one of the few still operational) you can see the MPD contains
````
<AdaptationSet id="3" group="3" contentType="text" lang="de" subsegmentAlignment="true"
   subsegmentStartsWithSAP="1" mimeType="application/mp4" codecs="wvtt" startWithSAP="1">
 <Role schemeIdUri="urn:mpeg:dash:role:2011" value="subtitle" />
 <Representation id="textstream_deu=1000" bandwidth="1000">
  <BaseURL>tears-of-steel-de.cmft</BaseURL>
  <SegmentBase timescale="1000" indexRangeExact="true" indexRange="607-1778">
   <Initialization range="0-606" />
  </SegmentBase>
 </Representation>
</AdaptationSet>
````

The `lang="de"`, `contentType="text"`, `mimeType="application/mp4"` and `codecs="wvtt"` are what led me to the given `<tva:Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:4"/>  <!--WebVTT-->` and `<tva:Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:3"/>  <!--ISOBMFF-->`